src/libs/snapshots.c

```diff
@@ -479,8 +479,10 @@ static void _clear_snapshot_entry(dt_lib_snapshot_t *s)
 
   g_free(s->module);
   g_free(s->label);
+  dt_free_align(s->buf);
```

Comment:
I'm pretty sure you still need `if(s->buf)` here as `_clear_snapshot_entry` is called to initialized the struct in `gui_init`.

Reply:
`dt_free_align(NULL)` is a no-op, like the other frees.

Reply:
Indeed, a wrong recollection on my side, sorry for the noise.